### PR TITLE
add luarocks for skywalking-nginx-lua.

### PR DESCRIPTION
**Pull request by moonming**
luarocks is package manager for lua

### CODE REVIEW

**Comment by wu-sheng** [comment]
Is this for release the package to luarock website?

**Comment by moonming** [comment]
yes, users can run `luarocks install skywalking-nginx-lua` after upload to luarocks website

**Comment by wu-sheng** [comment]
How could we apply the upload? Is there any doc or asf process?

**Comment by moonming** [comment]
We need to register an account first, then upload. Here are some similar distribution policies: https://cwiki.apache.org/confluence/display/INCUBATOR/DistributionGuidelines. 

The asf infra team now only support Github, Dockerhub, and Maven. 
NPM, pip and luarocks are need to handle by ourself.

**Review by wu-sheng — APPROVED** [review]

File: skywalking-nginx-lua-1.0-0.rockspec [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package = "skywalking-nginx-lua"
+version = "1.0-0"
+source = {
+   url = "git://github.com/apache/skywalking-nginx-lua",
+   branch = "master",
```
Comment (membphis) [inline skywalking-nginx-lua-1.0-0.rockspec:5]: We'd better use a separate version branch, or git tag.

Comment (wu-sheng) [reply skywalking-nginx-lua-1.0-0.rockspec:5]: Typically, SkyWalking will use tag to do release.

**Pull request merged**